app/assets/stylesheets/patternfly_and_overrides.scss

```diff
@@ -284,3 +284,11 @@ div.form-group .glyphicon-info-sign{
 .col-md-3 b{
   font-size: 100%;
 }
+
+span.status-error{
```

Comment:
This works well, but it's missing status-warn and status-question. Mind to group them all under the same css rule like:
```css
span.status-error, span.status-ok, span.status-warn, span.status-question {
  padding-left: 3px;
}
```